pennylane_lightning/core/src/lightning_gpu.toml

```diff
@@ -0,0 +1,125 @@
+schema = 1
+
+[device]
+name = "lightning.gpu"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+        "PauliX",
+        "PauliY",
+        "PauliZ",
+        "Hadamard",
+        "SparseHamiltonian",
```

Comment:
So we will include this after all, for the moment at least.

Reply:
Yes. If you want. Would you prefer it being removed?